docs-app/.prettierrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
       files: '*.hbs',
```

Comment:
Only added to `*.hbs` files for now as it doesn't seem to work yet with `.gts/gjs`